src/KendallQueue.java

```diff
@@ -0,0 +1,13 @@
+import lab1.KendallPacket;
+
+import java.util.ArrayDeque;
+import java.util.Queue;
+
+public class KendallQueue {
+
+    private Queue<KendallPacket> q;
+
+    public KendallQueue(int numPackets) {
+        this.q = new ArrayDeque<>(numPackets);
+    }
+}
```

Comment:
I was thinking of having KendallQueue be an interface, then have MD1Queue and MD1KQueue be subclasses of that. Then the interface would only have the private variable. MD1Queue would look like the current state of this class and MD1KQueue would override some of the queue methods.